### PR TITLE
MyVA | add forms 26-1880 and 28-8832 to SiP Form IDs set

### PR DESCRIPTION
**Pull request by DanielMiller-agile6**
## Summary
2 Forms were not showing up in the MyVA benefit application drafts section:

https://www.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction

https://www.va.gov/housing-assistance/home-loans/request-coe-form-26-1880/introduction

- add 2 constants to SiP form ID set in platform constants file, along with other required entries
- looks like code handling the SiP list was changed fairly recently in a way that made documentation out of date, so these 2 forms may have been missed. ticket has already been created for platform docs. adding them to the SIP_ENABLED_FORMS list should allow us to retest in staging and resolve.
- Team: MyVA / AuthExp Team

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/78504

## Testing done

- the 2 forms wouldn't show in the benefits application drafts section of MyVA
- verified via mocks + redux locally
- added e2e regression test.
- looks like these components don't have unit tests to build off of, so cypress seemed more efficient for now to solve the user's problem.
- we've had a few conversations with devs and designers on future work we might be able to do to improve things, but still requires more discovery
- visually tested with stakeholders, inspected visually w/ code inspection.
- used mock data inside of cypress manual runner.
- will test with stakeholders again on staging

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

**Before**:
Screenshot 2024-04-11 at 9 44 22 AM

___

**After**:
Screenshot 2024-04-11 at 8 08 55 AM

## What areas of the site does it impact?

MyVa UI & Platform JS code

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

~- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user~
No, I've had off/on difficulty with the full-stack local setup, and didn't find a test user with required SiP Form data yet. We will test on staging when available so stakeholders can re-test

## Requested Feedback
any/all

### CODE REVIEW

File: src/applications/personalization/dashboard/components/benefit-application-drafts/ApplicationInProgress.jsx [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { recordDashboardClick } from '~/applications/personalization/dashboard/h
 
 import DashboardWidgetWrapper from '../DashboardWidgetWrapper';
 
+const formatTitle = (title = '') => capitalize(title).replace(/\bva\b/, 'VA');
```
Comment (allisonlu) [inline src/applications/personalization/dashboard/components/benefit-application-drafts/ApplicationInProgress.jsx:9]: should we move this to `utils/helpers.js`?

**Review by allisonlu — APPROVED** [review]
🎉 looks great daniel!! i left one non launch-blocking comment. but otherwise :shipit:

File: src/applications/personalization/dashboard/components/benefit-application-drafts/ApplicationInProgress.jsx (hunk 1 above)
Comment (DanielMiller-agile6) [reply src/applications/personalization/dashboard/components/benefit-application-drafts/ApplicationInProgress.jsx:9]: sure :) we could if you think there's other use-cases that might need something similar?

**Review by adamwhitlock1 — APPROVED** [review]

**Review by powellkerry — APPROVED** [review]

File: src/applications/personalization/dashboard/components/benefit-application-drafts/ApplicationInProgress.jsx (hunk 1 above)
Comment (DanielMiller-agile6) [reply src/applications/personalization/dashboard/components/benefit-application-drafts/ApplicationInProgress.jsx:9]: i'm going to think on this for a future ticket and merge PR first so we can test on staging. I'm thinking maybe a generic "ensure VA remains capitalized in string" helper util could also be useful as well 🤔

**Pull request merged**